### PR TITLE
add new Activation function: selu

### PR DESCRIPTION
**Pull request by cc786537662**
add new Activation function :  selu

#read paper to know more detail:
    Self-Normalizing Neural Networks:https : //arxiv.org/abs/1706.02515
#discussion:
    https://www.reddit.com/r/MachineLearning/comments/6g5tg1/r_selfnormalizing_neural_networks_improved_elu/

#usage:
just like relu:
model.add(Dense(64, activation='relu'))
---model.add(Dense(64, activation='selu'))

experiment on mnist：（run keras-master\examples\mnist_cnn.py）：
with relu：
    Test loss: 0.025632923909
    Test accuracy: 0.9915
replace relu in network with selu:
    Test loss: 0.052635348707
    Test accuracy: 0.9837

### CODE REVIEW

**Comment by cc786537662** [comment]
forget to add some test in backend_test.py , PR  closed.

**Comment by ahundt** [comment]
@cc786537662 You might want to look at https://github.com/fchollet/keras/pull/6969 before you add more changes

**Comment by Danielhiversen** [comment]
@ahundt I think you mean https://github.com/fchollet/keras/pull/6924

**Comment by cc786537662** [comment]
nice job, that covers my PR,
thank you guys

**Comment by ahundt** [comment]
Yeah that's what I meant thanks thanks

**Comment by edmondja** [comment]
I added this to activations.py, and it seems to work fine with Theano.

def selu(x):
    scale = 1.0507009873554804934193349852946
    alpha = 1.6732632423543772848170429916717
    return scale * K.elu(x, alpha)

Is it less efficient than your method ?
